evaluator/this.go

```diff
@@ -10,5 +12,11 @@ func evaluateThis(node *ast.This, scope *object.Scope) object.Object {
 		return scope.Self
 	}
 
-	return object.NewError("%d:%d:%s: runtime error: cannot call 'this' outside of scope", node.Token.Line, node.Token.Column, node.Token.File)
+	pairs := make(map[object.MapKey]object.MapPair)
+
+	fmt.Printf("self: %v\n", scope.Self)
+
```

Comment:
This probably doesn't need to remain here..

Reply:
Thanks for catching this! This has been cleaned up.